backend/src/sms/routes/sms-settings.routes.ts

```diff
@@ -95,4 +102,37 @@ router.get(
   SettingsMiddleware.getChannelSpecificCredentials
 )
 
+/**
+ * @swagger
+ * path:
+ *  /settings/sms/credentials/verify:
+ *    post:
+ *      summary: Verify stored credential for user
+ *      tags:
+ *        - Settings
+ *      requestBody:
+ *        required: true
+ *        content:
+ *          application/json:
+ *            schema:
+ *              type: object
+ *              properties:
+ *                recipient:
+ *                  type: string
+ *                label:
+ *                  type: string
+ *
+ *      responses:
+ *        200:
+ *          description: OK
+ *        400:
+ *          description: Bad Request (invalid credentials, malformed request)
+ *
+ */
+router.post(
+  '/credentials/verify',
+  celebrate(verifyCredentialValidator),
+  SmsMiddleware.verifyUserCredential
```

Comment:
```suggestion
  SmsMiddleware.getCredentialsFromLabel,
  SmsMiddleware.validateAndStoreCredentials,
  (_req: Request, res: Response) => res.sendStatus(200)
```

Reply:
Changed. Thanks for pointing out the reuse of `validateAndStoreCredential` 🙏